### DISCUSSION TITLE
塞下曲·饮马渡秋水@王昌龄·唐

### DISCUSSION THREAD

**Discussion by aspoem-giscus** [meetqy/aspoem#6] [Announcements]
# 塞下曲·饮马渡秋水@王昌龄·唐

饮马渡秋水，水寒风似刀。
平沙日未没，黯黯见临洮。
昔日长城战，咸言意气高。
黄尘足今古，白骨乱蓬蒿。 

https://aspoem.com/poem/14

**Comment by meetqy** [comment]
这首乐府曲是以长城为背景，描绘战争的悲惨残酷。诗的前四句写塞外晚秋时节，平沙日落的荒凉景象；后四句写长城一带，历来是战场，白骨成丘，景象荒凉。全诗写得触目惊心，表达了非战思想。

**Comment by DUWENINK** [comment]
白bái骨gǔ乱luàn蓬péng蒿
